docker-compose.build.yml

```diff
@@ -12,7 +12,7 @@ services:
   ########################################################################
 
   sysmon-generator:
-    image: grapl/sysmon-generator:${TAG:-latest}
+    image: ${REGISTRY:-grapl}/sysmon-generator:${TAG:-latest}
```

Comment:
I think we'll want this to be: `${REGISTRY:-registry-1.docker.io}/grapl/sysmon-generator:${TAG:-latest}`, where registry-1.docker.io is Docker's default registry. This way the image name will still be prefixed with "grapl/".